src/osemgrep/cli_scan/Output.ml

```diff
@@ -43,7 +44,7 @@ let apply_fixes (conf : Scan_CLI.conf) (cli_output : Out.cli_output) =
   let edits : Textedit.t list =
     Common.map_filter
       (fun (result : Out.cli_match) ->
-        let path = result.Out.path in
+        let path = !!(result.Out.path) in
```

Comment:
I'm not a huge fan of mysterious operators like this one. It makes the codebase harder to onboard onto (in fact as a reviewer I don't actually know for sure what it does, I just guessed). Not expecting us to change it now, just mentioning it

Reply:
Not a fan in general, but I have to admit this one, which was designed by @mjambon, is really useful to migrate code from using Common.filename to Fpath.t. Without it the refactoring is really annoying, and in the long term everything will be an fpath so you will need far less of those !!

Reply:
Thanks for the context! That makes sense